### ISSUE TITLE
Is the code for a ATTINY45 or ATTINY44?

### ISSUE THREAD

**Issue by steves2j** [jacky4566/TinyParticleSupervisor#1]
Hi,

The code doesn't seem to compile OOTB for a ATTINY45 but it does for a ATTINY44. I had to change the PCMSK0,PCIE0,WDTCSR to PCMSK,PCIE and WDTCR before it compiles for a 45.

Wondering if this is just because of a recent change to the ATTinyCore or that the wrong version was pushed to git?

Which version of ATTinyCore did you build against?

**Comment by jacky4566** [comment]
Sorry @steves2j I decided to change my target MCU to the TINY44. Since both 44 and 45 come in the same QFN-20 package I changed my project for the extra IO.

I was also wanting better low power sleep on the boron so now my tiny controls the direct LIPO+ connection to the boron instead of the EN pin. Through a power switch of course. 

Ill try to get this project updated this weekend. 

Perhaps we can support both chips with some #ifdef
